client/common/cmdline.c

```diff
@@ -779,7 +777,7 @@ static char** freerdp_command_line_parse_comma_separated_values_offset(
 
 	p = t;
 
-	if (count)
+	if (*count)
```

Comment:
This looks wrong. The old version did check for NULL pointers, this dereferences...